src/library/ButtonGroup/ButtonGroup.js

```diff
@@ -195,7 +195,7 @@ const styles = ({ fullWidth, theme: baseTheme, variant }) => {
 // production, as long as it is set statically on each component
 // https://github.com/facebook/react/issues/4915#issuecomment-335803765
 const isButtonComponent = (element: React$Element<*>) => {
-  return /Button/.test(element.type.displayName);
+  return element.type ? /Button/.test(element.type.displayName) : false;
```

Comment:
I'm not sure why this is only necessary now. Because of the way `isButtonComponent` is called as part of `findDeep` ([line 287]()), sometimes the `element` here is `'[text]'`, which doesn't have a `type` property.

Reply:
I guess that makes sense since technically, elements can be strings.  Anyway, you could prolly rewrite this as...

```js
const isButtonComponent = (element: React$Element<*>) =>
  element.type && /Button/.test(element.type.displayName);
```